src/lib/y2ntp_client/widgets/main_widgets.rb

```diff
@@ -125,7 +133,7 @@ def help
           "Select whether to start the NTP daemon now and on every system boot. \n"            \
           "Selecting <b>Synchronize without Daemon</b> the NTP daemon will not be activated\n" \
           "and the system time will be set periodically by a <i>cron</i> script. \n"           \
-          "The interval is configurable, by default it is %d minutes.</p>"
+          "The interval is configurable, by default it is %d minutes."
         ) % Yast::NtpClientClass::DEFAULT_SYNC_INTERVAL
```

Comment:
You could fix it by adding it later, `(_(...) % Yast::NtpClientClass::DEFAULT_SYNC_INTERVAL) + "</p>"`.

On the other hand the translators might have already fixed that in the translations. So if the help text is displayed correctly I'd probably leave it as it is.

Reply:
Yep, I plan to fix only in master